### PR TITLE
Cleanup of main readme, remove digikey BOM and amazon list

### PR DESCRIPTION
**Pull request by Achllle**
Made a number of small fixes to the main readme I grouped in this PR. All of the commits are standalone so easy to revert any.

The build folder image is outdated so I removed it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,8 +122,6 @@ This project assumes you have some standard tools to help assemble the project.
  * Hand Drill or Drill Press
  * Dremel, Band saw, or hand saw
  * Items for operating a Raspberry Pi (Keyboard, mouse, monitor, 5V micro USB power adapter)
- 
- Here is [an Amazon list of some of the tools that may be useful](http://a.co/gnoN2LW)
```
Comment (apollokit) [inline README.md:130]: The amazon list includes wire crimpers. I'm fine with removing the link to the amazon list, but the crimpers ought to be added under optional tools

Comment (apollokit) [reply README.md:130]: https://www.amazon.com/dp/B07DWVSR41/?coliid=ISMG8XMBILGFK&colid=1IM7RHK25RIBB&psc=0&ref_=lv_vv_lig_dp_it was the item

File: README.md [hunk 2]
```diff
@@ -144,11 +138,6 @@ In our experience, this project takes no less than 200 person-hours to build, an
 
 ## Getting Started
 
-### Folder organization
-![Rover build folders](images/folders.png)
-
-The main /osr/ folder contains all of the documentation and information necessary for the project, broken down into the 3 main sections: Mechanical, Electrical, and Software. Each of these sections is meant to be relatively self contained and should be fairly parallelizable, meaning that they could be completed simultaneously by different groups. There is a README.md in each section to help you navigate the information in that section.
```
Comment (apollokit) [inline README.md:150]: i'm fine with getting rid of the folder organization image, but I think it's still useful to discuss the three main sections and that they're self contained and parallelizable. I'd leave some of this in

**Review by apollokit — CHANGES_REQUESTED** [review]
Some small change requests

I agree wrt the digikey BOM that it's not good to have two sources of truth. Though at the end of the day I want an easy way to order parts from digi-key. Hoping that https://github.com/nasa-jpl/open-source-rover/pull/180 might help with that.

File: README.md [hunk 3]
```diff
@@ -126,8 +122,6 @@ This project assumes you have some standard tools to help assemble the project.
  * Hand Drill or Drill Press
  * Dremel, Band saw, or hand saw
  * Items for operating a Raspberry Pi (Keyboard, mouse, monitor, 5V micro USB power adapter)
```
Comment (Achllle) [inline README.md:124]:
```suggestion
 * Items for operating a Raspberry Pi (Keyboard, mouse, monitor, 5V micro USB power adapter)
 * Wire strippers, e.g. [these](https://www.amazon.com/dp/B07DWVSR41/?coliid=ISMG8XMBILGFK&colid=1IM7RHK25RIBB&psc=0&ref_=lv_vv_lig_dp_it)
```

File: README.md (hunk 1 above)
Comment (Achllle) [reply README.md:130]: Done - FYI those aren't crimpers, they're strippers. Crimpers are for crimps :p

File: README.md (hunk 2 above)
Comment (Achllle) [reply README.md:150]: That's mentioned in [the buildmap section](https://www.amazon.com/dp/B07DWVSR41/?coliid=ISMG8XMBILGFK&colid=1IM7RHK25RIBB&psc=0&ref_=lv_vv_lig_dp_it) as well. If that doesn't suffice, can you use the suggestion feature to add a blurb you think makes sense?

**Comment by Achllle** [comment]
@apollokit @ericjunkins can you guys give this a quick check?

File: README.md (hunk 1 above)
Comment (apollokit) [reply README.md:151]: Indeed.

**Review by apollokit — APPROVED** [review]
Looks good

**Pull request merged**